recipe/meta.yaml

```diff
@@ -2,19 +2,20 @@
 {% set version = "0.55.0" %}
 
 {% set name = goname.split('/')[-1] %}
-{% set pkg_src = ('src/'+goname).replace("/",os.sep) %}
+{% set pkg_src = ('src/'+goname).replace("/",os.sep) %}  # src/github.com/junegunn/fzf
 
 package:
   name: {{ name|lower }}
   version: {{ version|replace("-", "_") }}
 
 source:
-  - folder: {{ pkg_src }}
-    url: https://{{ goname }}/archive/v{{ version }}.tar.gz
-    sha256: 805383f71bca7f8fb271ecd716852aea88fd898d5027d58add9e43df6ea766da
+  folder: {{ pkg_src }}
+  url: https://{{ goname }}/archive/v{{ version }}.tar.gz
+  sha256: 805383f71bca7f8fb271ecd716852aea88fd898d5027d58add9e43df6ea766da
 
 build:
   number: 0
+  skip: true  # [s390x]
   script:
     - pushd {{ pkg_src }}
     - go build -ldflags "-X main.revision=conda-forge" -v -o $PREFIX/bin/fzf  # [not win]
```

Comment:
instead of ../../../../library_licenses, using a directory env variable would be better I think